File: app/src/main/java/com/klinker/android/twitter_l/data/roomdb/pojos/TweetWithUserInfo.java
package com.klinker.android.twitter_l.data.roomdb.pojos;

import org.jetbrains.annotations.NotNull;

import androidx.room.Embedded;
import com.klinker.android.twitter_l.data.roomdb.entities.Tweet;
import com.klinker.android.twitter_l.data.roomdb.entities.User;


public class TweetWithUserInfo {

    @Embedded
    public Tweet tweet;

    @Embedded
    public User user;

}

File: app/src/main/java/com/klinker/android/twitter_l/data/roomdb/pojos/TweetWithUserInfo.kt
package com.klinker.android.twitter_l.data.roomdb.pojos

import androidx.room.Embedded
import com.klinker.android.twitter_l.data.roomdb.entities.Tweet
import com.klinker.android.twitter_l.data.roomdb.entities.User


class TweetWithUserInfo {

    @Embedded
    var tweet: Tweet? = null

    @Embedded
    var user: User? = null

}
